doc/getting_started/toolchain_custom_cmake.rst

```diff
@@ -3,25 +3,29 @@
 Custom CMake Toolchains
 #######################
 
-To use a custom toolchain defined in an external CMake file, export the
-following environment variables:
+To use a custom toolchain defined in an external CMake file, :ref:`set these
+environment variables <env_vars>`:
 
-.. code-block:: console
+- Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to your toolchain's name
+- Set :envvar:`TOOLCHAIN_ROOT` to the path to the directory containing your
+  toolchain's CMake configuration files.
+
+Zephyr will then include the toolchain cmake files located in the
+:file:`TOOLCHAIN_ROOT` directory:
 
-   # Linux and macOS
-   export ZEPHYR_TOOLCHAIN_VARIANT=<toolchain name>
-   export TOOLCHAIN_ROOT=<path to toolchain>
+- :file:`cmake/toolchain/generic.cmake`: configures the toolchain for "generic"
+  use (mostly to run the C preprocessor on the generated :ref:`device-tree`
+  file).
+- :file:`cmake/toolchain/target.cmake`: configures the toolchain for use
```

Comment:
Extra "use".

Reply:
Nope. Maybe you're reading "use" as a verb here, not a noun?

Reply:
Hmmm... what does "configures the toolchain for use building Zephyr" mean?  I think the "use" should be deleted too.

Reply:
It means configuring the toolchain so that it may be used to build zephyr applications...

Reply:
*This* "configure" I understand :-)